README.md

```diff
@@ -129,13 +129,26 @@ The default configuration assumes `mainnet` with only `btc` chain supported, and
 You can change the location of the configuration file with the `--conf=pathToConf` command line argument.
 
 ### Command line parameters
+Please note that NBXplorer uses cookie authentication by default. If you run your Bitcoin/Litecoin/Dash nodes using their daemon (like `bitcoind`, `litecoind` or `dashd`), they generate a new cookie every time you start them, and that should work without any extra configuration. 
+If you run the node(s) using the GUI versions, like Bitcoin\Litecoin\Dash Core Qt with the `-server` parameter while you set the rpcusername and rpcpassword in their `.conf` files, you must set those values for every crypto you are planning to support. 
+See samples below.
 
-#### From Source (.NET Core SDK required)
-The same settings as above, e.g.
-```dotnet run NBXplorer.dll --port=20300 --network=mainnet --nodeendpoint=127.0.0.1:32939```
+#### Run from source (.NET Core SDK required)
+```dotnet run NBXplorer.dll <parameters>```
+
+#### Run using built DLL (.NET Core Runtime required)
+```dotnet NBXplorer.dll <parameters>```
+
+#### Sample parameters
+Running NBXplorer HTTP server on port 20300, connecting to the BTC mainnet node locally.
+```--port=20300 --network=mainnet --nodeendpoint=127.0.0.1:32939```
+
+Running NBXplorer on testnet, supporting Bitcoin, Litecoin and Dash, using cookie authentication for BTC and LTC, and RPC username and password for Dash, connecting to all of them locally. 
+```--chains=btc,ltc,dash --network=testnet --dashrpcuser=myuser --dashrpcpassword=mypassword```
+
+Running NBXplorer with a BTC rescan from January 1st of 2019.
```

Comment:
This should never be needed though, I added this in case of bug.

Reply:
I can add more information about this case, but I still think it's useful for testing. 
If I understand it correctly with this we can let NBXplorer know that there were transactions before it's starting time that are important regarding to the wallet I'm tracking. 
Although rarely, but this can happen.

Reply:
Well, in this case my advise is to use the scanutxoset API that NBXplorer expose. This is way faster than a rescan and does not require a restart of the process.

Reply:
Alright, thanks for the explanation!
I remove this case then.